py-polars/polars/_utils/construction/series.py

```diff
@@ -422,9 +422,10 @@ def pandas_to_pyseries(
             "(e.g. 'int64', 'bool', 'float32' - not 'Int64')"
         )
         raise ImportError(msg)
-    return arrow_to_pyseries(
+    s = arrow_to_pyseries(
         name, plc.pandas_series_to_arrow(values, nan_to_null=nan_to_null)
     )
+    return s if dtype is None else s.cast(dtype, strict=False)
```

Comment:
is it right to hardcode `strict=False` here?

Reply:
Thank you for the review.

The hard-coded form shouldn't be used. I have made modifications based on references to other functions in the same file.